.travis.yml

```diff
@@ -19,13 +19,14 @@ before_install:
 - export PATH=$PATH:~/.bin
 install:
 - pip install -r requirements.txt
-- pip install coveralls
+- pip install coveralls pep8
```

Comment:
@hackaugusto Then maybe remove the pep8 tool invocation from here until https://github.com/PyCQA/pycodestyle/pull/502 is merged?

No reason to leave this PR hanging on our side. It does fix a lot of style issues.